docs/Displaying a User Interface/Main Menu/index.md

```diff
@@ -8,7 +8,33 @@ You have two different options when creating menus. One is to simply add items t
 Every template has a main menu button. The position of this button varies between templates and cannot be removed from the template. Some OEMs may format certain templates to not display the main menu button if you have no menu items (such as the navigation map view).
 !!!
 
-## Adding Menu Items
+## Setting a Menu Style
```

Comment:
Move the **Menu Template** header and screenshot that is currently above this section to the **Setting the Menu Layout** section. Also add a generic hmi screenshot of the same menu with the tile format.